apis/python/src/tiledb/vector_search/object_api/embeddings_ingestion.py

```diff
@@ -182,16 +178,15 @@ def instantiate_object(code, class_name, **kwargs):
                 return class_(**kwargs)
 
             logger = setup(config, verbose)
-            object_reader = instantiate_object(
-                code=object_reader_source_code,
-                class_name=object_reader_class_name,
-                **object_reader_kwargs,
-            )
-            object_embedding = instantiate_object(
-                code=object_embedding_source_code,
-                class_name=object_embedding_class_name,
-                **object_embedding_kwargs,
+            ob_index = object_index.ObjectIndex(
```

Comment:
(Minor)(Optional) Maybe could call this `obj_index`? `ob` is a bit hard for me to understand without thinking about it a bit 😅

Reply:
Applied